db/migrate/20150608190453_create_results.rb

```diff
@@ -0,0 +1,17 @@
+class CreateResults < ActiveRecord::Migration
+  def change
+    create_table :results do |t|
```

Comment:
I think we can focus on the data model validations only for now. And circle back around to the user facing stuff on that other card we already have.